e2etest/local_transfer_test.go

```diff
@@ -603,118 +603,88 @@ func lenPendingActionMap(acts map[string][]action.SealedEnvelope) int {
 }
 
 type chainCase1 struct {
-	cfgChainID       uint32
-	cfgJutlandHeight uint64
-	testChainID      uint32
-	err              error
+	testChainID uint32
+	success     int
 }
 
 func TestChainIDWithJutland(t *testing.T) {
 
 	testCase := []chainCase1{
 		{
-			1, 30, 1, nil, // tx chainID = node chainID, height < jutland
+			1, 1, // tx chainID = node chainID, height < KamchatkaHeight
 		},
 		{
-			2, 31, 2, nil, // tx chainID = node chainID, height >= jutland
+			1, 1, // tx chainID = node chainID, height >= KamchatkaHeight
 		},
 		{
-			4, 31, 2, nil, // tx chainID != node chainID, height < jutland
+			1, 1, // tx chainID != node chainID, height < KamchatkaHeight
 		},
 		{
-			2, 31, 4, nil, // tx chainID != node chainID, height < jutland
+			1, 1, // tx chainID != node chainID, height < KamchatkaHeight
 		},
 		{
-			4, 30, 2, action.ErrChainID, // tx chainID != node chainID, height >= jutland
+			2, 0, // tx chainID != node chainID, height >= KamchatkaHeight
 		},
 		{
-			2, 30, 4, action.ErrChainID, // tx chainID != node chainID, height >= jutland
+			2, 0, // tx chainID != node chainID, height >= KamchatkaHeight
 		},
 		{
-			4, 10, 2, action.ErrChainID, // tx chainID != node chainID, height >= jutland
+			2, 0, // tx chainID != node chainID, height >= KamchatkaHeight
 		},
 		{
-			2, 10, 4, action.ErrChainID, // tx chainID != node chainID, height >= jutland
+			2, 0, // tx chainID != node chainID, height >= KamchatkaHeight
```

Comment:
1. all test case with height < KamchatkaHeight should be before height > KamchatkaHeight, right?
2. test case does not cover `2 != 1 && height < KamchatkaHeight`, the tx can be committed

Reply:
to be specific, test case should be:
```
KamchatkaBlockHeight = uint64(3)
chainID = 1, success, // commit at height 1
chainID = 2, success, // commit at height 2
chainID = 1, success, // commit at height 3 = Kamchatka
chainID = 2, fail, // commit at height 4 > Kamchatka
```